JGActionSheet/JGActionSheet.h

```diff
@@ -248,6 +248,26 @@ typedef NS_ENUM(NSUInteger, JGActionSheetArrowDirection) {
  */
 - (void)dismissAnimated:(BOOL)animated;
 
+/** Shows the action sheet anchored to a specific view based on the arrow direction
+ @param anchorView the view that the sheet will be anchored to and the position point will be calculated from
+ @param parentView the hosting view in which the action sheet should be shown
+ @param arrowDirection The direction in which the arrow should show and the side of the section at which the arrow should be placed.
+ @param animated Whether the action sheet should show with an animation.
+
+ @Attention This method is only available on iPad devices.
+ */
+
+- (void)showAtView:(UIView *)anchorView inView:(UIView *)displayView withArrowDirection:(JGActionSheetArrowDirection)arrowDirection animated:(BOOL)isAnimated;
+
+/** These two methods set variables based on whether or not the parent view of the cell is a table view or collection view
```

Comment:
Why expose this? Can’t the cell look at its parent view and figure out what to do? (Perhaps it can’t, in which case explaining that here would be good.)